lib/plan_picker/subject.ex

```diff
@@ -20,6 +21,17 @@ defmodule PlanPicker.Subject do
     |> Repo.insert!()
   end
 
+  def get_subject_in_enrollment!(enrollment, subject_id, opts \\ [preload: [classes: [:terms, :teacher, :users]]]) do
```

Comment:
1. What does this function do? 
2. Why we're passing enrollment struct and subject id instead of two structs?

Reply:
Was overly complex, fixed